keyboards/keebio/iris/keymaps/jasonkrasavage/keymap.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
```

Comment:
**Optional, but recommended:**

Use an enum for managing your layer indexes, by replacing Lines 5-8 with:

```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```

This way future layer changes only require modifying the enum, rather than a series of indexes.

Reply:
Hey @noroadsleft, I will make that change. That's an elegant solution. This is actually just built off of the default keymap.c for the iris and it was like that in the beginning. I have only changed some of the keys I needed to change and haven't gone in deeper to optimize anything. I will wait for the pull request for my new folder to be accepted and start making further changes after that. Thanks a lot for looking!

Reply:
Did you make the changes on your hard drive and then forget to push to GitHub? I don't see any additional changes beyond your original five commits.

Reply:
There was already an enum in there which appeared to be redundant. Just pushed the changes tho. These issues and changes should really be pushed into the original default keymap folder since this file (keymap.c) is essentially that exact default keymap file with a few key codes altered. That way anyone new building off of the default mapping doesn't have to also make these changes. Is that something I should do or is there someone else already working on the default iris mapping?